src/components/Common/Image.tsx

```diff
@@ -0,0 +1,9 @@
+import { ImgHTMLAttributes } from 'react';
+
+type ImageProps = ImgHTMLAttributes<HTMLImageElement>;
+
+function Image({ ...props }: ImageProps) {
+  return <img className="object-cover w-full h-full" {...props} loading="lazy" />;
+}
```

Comment:
(마이너) HTMLImageElement 속성에 포함되어있으니 기본값으로 설정해주면 어떨까요? 
```suggestion
function Image({ loading = 'lazy', ...props }: ImageProps) {
  return <img className="object-cover w-full h-full" {...props} loading={loading} />;
}
```

Reply:
@EungyuCho 

아! 그게 더 좋겠네요!